bylaws.md

```diff
@@ -1,51 +1,46 @@
 # Navn, formål, hjemsted og aktiviteter
-Foreningens navn er The Oeresund Local Chapter of Association for Computing Machinery, og anvender også binavnene ACM Øresund, ACM Öresund or ACM ø̈resund.
+Foreningens navn er The Oeresund Chapter of Association for Computing Machinery, og anvender også binavnene ACM Øresund og ACM Öresund. The Association for Computing Machinery benævnes herefter ACM.
```

Comment:
Need to check for hard requirements https://www.acm.org/chapters/chapter-policies